Migration.md

```diff
@@ -48,6 +48,10 @@ message's header.
       * `GZ_<PROJECT>_<VISIBLE/HIDDEN>`
       * CMake `-config` files
       * Paths that depend on the project name
+
+  * The `Scene3D` plugin has been removed and replaced with `gz-gui`'s `MinimalScene` plugin. See
+    this same document for the instructions to replace it when it was deprecated 5.x to 6.x.
```

Comment:
>The instructions for this is under ## Gazebo Sim 6.1 to 6.2 not 5.x to 6.x

Sound weird to me that we have removed a plugin in a minor bump.

Reply:
> It may be good to add that setting <legacy>false</legacy> is no longer required (removed in https://github.com/gazebosim/gz-sim/pull/1605) since we're redirecting them to the instructions below

Done in [799f7ac](https://github.com/gazebosim/gz-sim/pull/1698/commits/799f7ac6a89f8b121ad539947a156638426d6133)

Reply:
> > The instructions for this is under ## Gazebo Sim 6.1 to 6.2 not 5.x to 6.x
> 
> Sound weird to me that we have removed a plugin in a minor bump.

I was mistaken, I thought the content was in the wrong spot.